lib/iris/tests/unit/cube/test_Cube.py

```diff
@@ -1817,6 +1817,17 @@ def test_numerical_tolerance(self):
         self.assertAlmostEqual(result.coord("longitude").points[0], 28.5)
         self.assertAlmostEqual(result.coord("longitude").points[-1], 67.5)
 
+    def test_numerical_tolerance_wrapped(self):
+        # test the tolerance on the coordinate value causes modulus wrapping
+        # where appropriate
+        cube = create_cube(0.5, 3600.5, bounds=True)
+        lons = cube.coord("longitude")
+        lons.points = lons.points / 10
+        lons.bounds = lons.bounds / 10
+        result = cube.intersection(longitude=(-60, 60))
+        self.assertAlmostEqual(result.coord("longitude").points[0], -60.05)
+        self.assertAlmostEqual(result.coord("longitude").points[-1], 60.05)
```

Comment:
@lbdreyer Could we explicitly test the bounds here within this test?

Previously, this might have helped us detect that all was not well e.g., something long the lines of:
```python
        lon = result.coord("longitude")
        self.assertAlmostEqual(lon.points[0], -60.05)
        self.assertAlmostEqual(lon.points[-1], 60.05)
        dtype = lon.bounds.dtype
        np.testing.assert_array_almost_equal(lon.bounds[0], np.array([-60.1, -60.], dtype=dtype))
        np.testing.assert_array_almost_equal(lon.bounds[-1], np.array([60., 60.1], dtype=dtype))
```
And perhaps also do the same for the previous test `test_numerical_tolerance` :+1: